spec/index.bs

```diff
@@ -307,12 +307,18 @@ dictionary GPUTextureViewDescriptor : GPUObjectDescriptorBase {
     required GPUTextureViewDimension dimension;
     required GPUTextureAspect aspect;
     u32 baseMipLevel = 0;
-    u32 mipLevelCount = 1;
+    u32 mipLevelCount = 0;
     u32 baseArrayLayer = 0;
-    u32 arrayLayerCount = 1;
+    u32 arrayLayerCount = 0;
 };
 </script>
 
+  * {{GPUTextureViewDescriptor/mipLevelCount}}:
+    If mipLevelCount == 0, baseMipLevel must also be 0 and the texture view will cover all the mipmap levels of the original texture.
```

Comment:
We don't actually need this restriction. We could say that the `mipLevelCount`, if 0, covers all the mips starting from the `baseMipLevel`

Reply:
Done